### DISCUSSION TITLE
Plans for further development of this library

### DISCUSSION THREAD

**Discussion by crejk** [neg4n/trickster#7] [Q&A] [answered]
Hey, are there any plans for further development of this library? The whole concept is interesting, but there is a lack of features implemented.

**Comment by neg4n** [comment, marked as answer]
Hello, the library itself was meant to be large set of tools for memory manipulation and also a nice exercise of Rust for me 😁
But then came the time issue or rather lack of it at all, and the project remained just the exercise, I doubt if anyone is actually using it.

Trickster might be continued when I will have free time and I will start focusing more on OSS but now it will remain as it is, unless repository maintainer will be found.

Best regards.
